control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator.go

```diff
@@ -446,7 +446,11 @@ kubectl --kubeconfig $kc config use-context default`,
 		},
 	}
 
-	if semver.MustParse(params.ReleaseVersion).Minor == uint64(12) {
+	parsedReleaseVersion, err := semver.Parse(params.ReleaseVersion)
+	if err != nil {
+		return fmt.Errorf("parsing ReleaseVersion (%s): %w", params.ReleaseVersion, err)
+	}
+	if parsedReleaseVersion.Minor == uint64(12) {
```

Comment:
Hey @csrwng do we still need this check? AFAIK IBM moved from 4.12 some time ago in all regions.

Reply:
Either way, I'd be inclined to separate a functional change to a different PR. This will make is easier to backport/revert them independently as required.